docs/src/devel/dev-env-setup.rst

```diff
@@ -178,3 +178,216 @@ e.g.:
+    os_migrate_conv_dst        : ok=12   changed=5    unreachable=0    failed=0    skipped=3    rescued=0    ignored=0
+    os_migrate_conv_src        : ok=12   changed=5    unreachable=0    failed=0    skipped=3    rescued=0    ignored=0
+
+    Wednesday 21 July 2021  09:59:17 +0000 (0:00:03.419)       0:29:17.016 ********
+    ===============================================================================
+    os_migrate.os_migrate.conversion_host_content : update all packages --------------------------------------- 435.56s
+    os_migrate.os_migrate.import_workloads : transfer volumes to destination ---------------------------------- 101.72s
+    os_migrate.os_migrate.import_workloads : expose source volumes --------------------------------------------- 66.67s
+    os_migrate.os_migrate.conversion_host_content : install content -------------------------------------------- 62.35s
+    os_migrate.os_migrate.import_workloads : transfer volumes to destination ----------------------------------- 58.75s
+    os_migrate.os_migrate.import_workloads : clean up in the source cloud after migration ---------------------- 27.40s
+    os_migrate.os_migrate.import_workloads : expose source volumes --------------------------------------------- 27.30s
+    Create osm_server ------------------------------------------------------------------------------------------ 24.71s
+    create osm_image ------------------------------------------------------------------------------------------- 23.86s
+    os_migrate.os_migrate.export_images : export image blobs --------------------------------------------------- 23.80s
+    os_migrate.os_migrate.import_images : import images -------------------------------------------------------- 23.69s
+    os_migrate.os_migrate.import_workloads : create destination instance --------------------------------------- 23.30s
+    os_migrate.os_migrate.import_workloads : create destination instance --------------------------------------- 21.93s
+    Create osm_server ------------------------------------------------------------------------------------------ 21.61s
+    os_migrate.os_migrate.import_workloads : clean up in the source cloud after migration ---------------------- 21.16s
+    os_migrate.os_migrate.conversion_host : create os_migrate conversion host ---------------------------------- 20.03s
+    Remove osm_server ------------------------------------------------------------------------------------------ 19.01s
+    os_migrate.os_migrate.conversion_host : create os_migrate conversion host ---------------------------------- 18.23s
+    Shutdown osm_server ---------------------------------------------------------------------------------------- 18.14s
+    Shutdown osm_server ---------------------------------------------------------------------------------------- 17.88s
+
+Optional tags to pass to e2e tests
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are a set of tags that can be used to filter which playbooks to run during test.
```

Comment:
I'd say "which playbooks" -> "which plays" or "which tasks". I'd go with the latter as it's more likely to be correct in any circumstances.

Reply:
Thank you